job_templates/ci_job.xml.em

```diff
@@ -60,7 +60,7 @@
     <submoduleCfg class="list"/>
     <extensions>
       <hudson.plugins.git.extensions.impl.SubmoduleOption>
-        <disableSubmodules>false</disableSubmodules>
+        <disableSubmodules>@[if avoid_credentials]true@[else]false@[end if]</disableSubmodules>
```

Comment:
I could see this becoming a problem if any other repo requires a submodule in the future...

Reply:
Nitpick: you can avoid the bracket syntax here: `@('true' if avoid_credentials else 'false')`.